engine/graphics/src/webgpu/graphics_webgpu.cpp

```diff
@@ -1018,7 +1023,7 @@ static HContext WebGPUNewContext(const ContextParams& params)
 static bool WebGPUIsSupported()
 {
     TRACE_CALL;
-    return true;
+    return isWebGPUSupported();
```

Comment:
Check variable from JS part instead of returning true everytime.

Reply:
FWIW I am currently using an implementation of native webgpu and it does indeed function, makes me sad to have to go back to JS for this.

Reply:
At least current implementation when returns true doesn't work correctly for browsers which don't support WebGPU. It's also other small bug in how graphics adapter is selected but by default WebGPU selected every time because has higher priority than OpenGL implementation.
Question about "use JS or not use" - I commented below. Let's wait for the input from the team to understand JS code should be avoided or not.